### DISCUSSION TITLE
Shorter board with all smd - is that possible?

### DISCUSSION THREAD

**Discussion by 8lall0** [frntc/SIDKick-pico#27] [Announcements]
First of all, kudos for the project: IS GREAT.

Second: in your opinion, would be possible to elaborate a pcb that can contain all the parts (shitfters + raspi pico stuff + dac) to solder on, to make it smaller?

**Comment by frntc** [comment]
Thanks!

Yes, I have designed such a PCB (not tested yet), but I cann tell you that this is nothing for hand soldering: plenty of really small footprint components. Nevertheless, I will continue this path, even if it's just to have it done ;-)

As a middle-ground there will be a PCB with the DAC and OpAmp on board.

**Reply by LJNielsenDk** [reply]
That middle-ground board would be nice.

**Reply by frntc** [reply]
Will look like this and be released together with a (major) firmware upgrade...

![SKpicoDAC](https://github.com/frntc/SIDKick-pico/assets/48584231/7f51477a-f714-4b0f-b3fa-5313ea56d743)

**Reply by 8lall0** [reply]
Looks very neat!
